### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by CristianRWolox**
## Summary

- Animate card opacity.
- Add animated icon for success rent.

#### Pixel 3XL
untitled-min

#### iPhone 11
https://user-images.githubusercontent.com/83975070/120649918-e55e9a00-c442-11eb-887a-475d505512fd.mp4

## Trello Card

[Trello Card - Animation](https://trello.com/c/Z2ntTyAg/34-n-13-animations)

### CODE REVIEW

File: wbooks/src/app/components/Button/index.tsx [hunk 1]
```diff
@@ -10,9 +10,9 @@ interface Props {
 
 const Information = ({ solid = true, text }: Props) => {
   return (
-    <TouchableOpacity style={solid ? styles.btnSolid : styles.btnHollow}>
+    <View style={solid ? styles.btnSolid : styles.btnHollow}>
```
Comment (mcolombo87) [inline wbooks/src/app/components/Button/index.tsx]: this not look good, it's a component of the all app named "Button" withOut touchable. I can see why you change this, but the component  name with your location not look good.

**Review by mcolombo87 — CHANGES_REQUESTED** [review]

File: wbooks/src/app/components/Button/index.tsx (hunk 1 above)
Comment (guidoprinc) [reply wbooks/src/app/components/Button/index.tsx]: If the component is named `Button` it is expected to have a 'pressable'  functionality

Comment (CristianRWolox) [reply wbooks/src/app/components/Button/index.tsx]: Ok, I get it. I'm going to left the touchable with a dummy onPress

File: wbooks/src/app/screens/BookDetail/components/Information/styles.ts [hunk 2]
```diff
@@ -37,5 +39,18 @@ export default StyleSheet.create<Styles>({
...
+    marginTop: 20,
+    alignSelf: 'center',
+    backgroundColor: COLORS.green,
+    alignItems: 'center',
+    borderRadius: 20,
+    padding: 5
+  },
+  successImage: {
+    height: 17,
+    width: 17,
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/Information/styles.ts]: extract in a common constant since it's a square size element

**Review by guidoprinc — CHANGES_REQUESTED** [review]

File: wbooks/src/app/screens/BookDetail/components/Information/styles.ts (hunk 2 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/BookDetail/components/Information/styles.ts]: Done!

**Review by guidoprinc — APPROVED** [review]

**Review by mcolombo87 — APPROVED** [review]

**Pull request merged**